atom/browser/api/atom_api_web_contents.cc

```diff
@@ -119,10 +126,57 @@ struct Converter<PrintSettings> {
       return false;
     dict.Get("silent", &(out->silent));
     dict.Get("printBackground", &(out->print_background));
+    dict.Get("deviceName", &(out->device_name));
     return true;
   }
 };
 
+using StringMap = std::map<std::string, std::string>;
+
+template<>
+struct Converter<StringMap> {
+    static v8::Local<v8::Value>
+    ToV8(v8::Isolate* isolate, const StringMap& val) {
+      v8::Local<v8::Object> result = v8::Object::New(isolate);
+      for (auto i = val.begin(); i != val.end(); i++) {
+        result->Set(StringToV8(isolate, i->first),
+                    StringToV8(isolate, i->second));
+      }
+      return result;
+    }
+};
+
+template<>
+struct Converter<printing::PrinterBasicInfo> {
+  static v8::Local<v8::Value>
+  ToV8(v8::Isolate* isolate, const printing::PrinterBasicInfo& val) {
+    v8::Local<v8::Object> result = v8::Object::New(isolate);
+    result->Set(StringToV8(isolate, "printerName"),
+                StringToV8(isolate, val.printer_name));
+    result->Set(StringToV8(isolate, "printerDescription"),
+                StringToV8(isolate, val.printer_description));
+    result->Set(StringToV8(isolate, "printerStatus"),
+                ConvertToV8(isolate, val.printer_status));
+    result->Set(StringToV8(isolate, "isDefault"),
+                ConvertToV8(isolate, val.is_default));
+    result->Set(StringToV8(isolate, "options"),
+                ConvertToV8(isolate, val.options));
+    return result;
+  }
+};
+
+template<>
+struct Converter<PrinterList> {
```

Comment:
This converter is not required since there is already `std::vector<T>` converter in `native_mate`.